README.md

```diff
@@ -91,75 +91,66 @@ As a root launch HAProxy with configuration:
 Load haproxy plugin:
 ```
-$ $SNAP_PATH/bin/snapctl plugin load snap-plugin-collector-haproxy
+$ $SNAP_PATH/bin/snapctl plugin load snap-plugin-collector-haproxy/build/rootfs/snap-plugin-collector-haproxy
 ```
 See available metrics for your system
 ```
 $ $SNAP_PATH/bin/snapctl metric list
 ```
-Create a task manifest file (exemplary file in [example/task/] (https://github.com/intelsdi-x/snap-plugin-collector-haproxy/blob/master/example/task/)):
-```json
+Create a task manifest file (exemplary file in [examples/tasks/] (examples/tasks/)):
+```
 {
-    "version": 1,
-    "schedule": {
-        "type": "simple",
-        "interval": "1s"
-    },
-    "workflow": {
-        "collect": {
-            "metrics": {
-                "/intel/linux/haproxy/info/CurrConns": {},
-                "/intel/linux/haproxy/info/Run_queue": {}, 
-                "/intel/linux/haproxy/stat/<service_name>/<proxy_name>/qlimit": {},
-                "/intel/linux/haproxy/stat/<service_name>/<proxy_name>/chkdown": {} 
-            },
-            "config": {
-                "/intel/mock": {
-                    "password": "secret",
-                    "user": "root"
-                }
-            },
-            "process": [
-                {
-                    "plugin_name": "passthru",
-                    "process": null,
-                    "publish": [
-                        {                         
-                            "plugin_name": "file",
-                            "config": {
-                                "file": "/tmp/published_haproxy"
-                            }
-                        }
-                    ],
-                    "config": null
-                }
-            ],
-            "publish": null
+  "version": 1,
+  "schedule": {
+    "type": "simple",
+    "interval": "1s"
+  },
+  "workflow": {
+    "collect": {
+      "metrics": {
+        "/intel/haproxy/*": {}
+      },
+      "config": null,
+      "process": [
+        {
+          "plugin_name": "passthru",
+          "process": null,
+          "publish": [
+            {
+              "plugin_name": "mock-file",
+              "config": {
+                "file": "/tmp/published_haproxy"
+              }
+            }
+          ]
         }
+      ]
     }
+  }
 }
+
 ```
 Load passthru plugin for processing:
 ```
-$ $SNAP_PATH/bin/snapctl plugin load build/plugin/snap-processor-passthru
+$ $SNAP_PATH/bin/snapctl plugin load $SNAP_PATH/plugin/snap-plugin-processor-passthru
 ```
 
-Load file plugin for publishing:
+Load mock file plugin for publishing:
 ```
-$ $SNAP_PATH/bin/snapctl plugin load build/plugin/snap-publisher-file
+$ $SNAP_PATH/bin/snapctl plugin load $SNAP_PATH/plugin/snap-plugin-publisher-mock-file
 ```
 Create a task:
 ```
-$ $SNAP_PATH/bin/snapctl task create -t examples/tasks/haproxy-file.json
+$ $SNAP_PATH/bin/snapctl task create -t snap-plugin-collector-haproxy/examples/tasks/haproxy-file.json
 Using task manifest to create task
```

Comment:
Let's consider removing SNAP_PATH as a requirement since this requires that the snap source has been installed and built.

To do this I propose removing the passthru plugin from the example and describe using curl to get the latest snap-plugin-publisher-file through http://snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-file/master/latest/snap-plugin-publisher-file-linux or http://snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-file/master/latest/snap-plugin-publisher-file-darwin.